src/inputs/button/buttonConstants.js

```diff
@@ -56,6 +61,10 @@ export const buttonPropTypes = {
      */
     style: PropTypes.shape({}),
     target: PropTypes.oneOf(['_blank']),
```

Comment:
NAB.  Should we add a documentation comment for `target` prop?

Reply:
fixed.

Reply:
Resolving.